internal-services/catalog/create-advisory-task.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: task-create-advisory
```

Comment:
Question maybe for @theflockers : Why is there a different naming for the file vs. metadata? i.e. blabla-task vs. task-blabla. Just curious how that happened of if there is a reason for that. (Not saying we need to change it or anything.)

Reply:
Yeah I just followed the others

Reply:
Yeah, if this thread is still unanswered when the PR is ready, just close it.

Reply:
I do not remember any reason unless I was following some other examples at that time....

Reply:
I changed the task name to `create-advisory-task` instead of `task-create-advisory`

Reply:
Hmm. The pipeline has pipeline in filename, but not name, but the task has task in both. Do we need it in the task name in the yaml? Probably not a big deal, since this means it will not be consistent with the rest of the repo anyway.